dask/dataframe/tests/test_groupby.py

```diff
@@ -118,15 +118,25 @@ def test_full_groupby_apply_multiarg():
                       index=[0, 1, 3, 5, 6, 8, 9, 9, 9])
     ddf = dd.from_pandas(df, npartitions=3)
 
-    pytest.raises(Exception, lambda: df.groupby('does_not_exist'))
-    pytest.raises(Exception, lambda: df.groupby('a').does_not_exist)
-    assert 'b' in dir(df.groupby('a'))
-
     def func(df, c, d=3):
         return df.assign(b=df.b - df.b.mean() + c * d)
 
-    assert_eq(df.groupby('a').apply(func, 1),
-              ddf.groupby('a').apply(func, 1))
+    c = 1
+    d = 2
+    c_scalar = dd.core.Scalar({'my-scalar': c}, 'my-scalar', int)
+    d_scalar = dd.core.Scalar({'my-scalar': d}, 'my-scalar', int)
```

Comment:
Rather than use internal API I recommend that we us something like `df.a.sum()`.  This will make this test less brittle to internal changes and make it easier for novice maintainers to understand.